astropy/coordinates/earth.py

```diff
@@ -20,6 +22,9 @@
 # Available ellipsoids (defined in erfam.h, with numbers exposed in erfa).
 ELLIPSOIDS = ('WGS84', 'GRS80', 'WGS72')
 
+_NO_ONLINE_SITES_WARNING_MSG = ('Could not access the online site list. Falling'
```

Comment:
Good point, will fix
I *usually* do that too but in this case was trying to avoid the 80 chars (silly, really, but it's sort of a little game...)